justbuild.it-WebApp/src/main/java/justbuild/it/web/app/controller/WebAppController.java

```diff
@@ -18,10 +20,12 @@ public class WebAppController {
 
     private final OfferService offerService;
     private final OfferMapper mapper;
+    private final OfferEditService offerEditService;
```

Comment:
to trzeba usunąć i wstrzykiwac klasę OfferEditionService do klasy OfferService, klasa która edytuje ofertę powinna byc package-private

Reply:
klasa OfferService mogłaby wyglądać tak, żeby zrobić to najszybciej :)

@Service
public class OfferService {

    private final OfferCreationService offerCreationService;
    private final OfferSearchingService offerSearchingService;
    private final OfferEditionService offerEditionService;

    public OfferService(OfferCreationService offerCreationService, OfferSearchingService offerSearchingService, OfferEditionService offerEditionService) {
        this.offerCreationService = offerCreationService;
        this.offerSearchingService = offerSearchingService;
        this.offerEditionService = offerEditionService;
    }
    
    // Twoje metody, które OfferService przekazuje do kontrolera
    
    public OfferDto getOfferDtoById(Long id) {
        return offerEditionService.getOfferDtoById(id);
    }

    public void updateOffer(OfferDto editedOfferDto) {
        offerEditionService.updateOffer(editedOfferDto);
    }